### PR TITLE
Make JetResolution class available from python

### PR DESCRIPTION
**Pull request by HeinerTholen**
Works like this in python:

    import ROOT
    res_obj = ROOT.JME.JetResolution('Spring16_25nsV10_MC_PtResolution_AK4PFchs.txt')
    jet_res = jet_pt * res_obj.getResolution(ROOT.JME.JetParameters().setRho(rho).setJetPt(jet_pt).setJetEta(jet_eta))

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: JetMETCorrections/Modules/src/classes_def.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<lcgdict>
+    <class name="JME::JetResolution" class_version="0"/>
+</lcgdict>
```
Comment (slava77) [inline JetMETCorrections/Modules/src/classes_def.xml]: an end line would be useful

File: JetMETCorrections/Modules/src/classes_def.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<lcgdict>
+    <class name="JME::JetResolution" class_version="0"/>
```
Comment (slava77) [inline JetMETCorrections/Modules/src/classes_def.xml]: is class_version needed?
I see that it's a common attribute in the CondFormats and nowhere else in CMSSW.

Perhaps ```persistent="false" ``` is  more useful.

Comment (HeinerTholen) [reply JetMETCorrections/Modules/src/classes_def.xml]: Hi Slava, I just copied from another classes_def.xml and I don't have an opinion about it. Will add ``persistent="false"`` and the end line soon.

**Comment by perrotta** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/22879	336c814
- technical change: dictionary specs were added for JME::JetResolution, in line with the PR description.
- jenkins tests pass.

**Comment by HeinerTholen** [comment]
@fabiocos When do you expect this PR to be available in a release and which release would that be?

**Comment by slava77** [comment]
@HeinerTholen 
this was merged 15 days ago and is available in the integration builds since then.

**Comment by HeinerTholen** [comment]
Can you point me to a release under /cvmfs?
E.g. /cvmfs/cms.cern.ch/slc6_amd64_gcc700/cms/cmssw/CMSSW_10_X_Y

Sorry if I don't quite get it..

**Comment by HeinerTholen** [comment]
@slava77 sorry, did you see my question above?

**Comment by slava77** [comment]
```scram list -c CMSSW``` is the standard way to discover the release location

```
export SCRAM_ARCH=slc6_amd64_gcc700
scram list -c CMSSW | grep 10_2_X
CMSSW           CMSSW_10_2_X_2018-04-15-0000 /cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc700/cms/cmssw/CMSSW_10_2_X_2018-04-15-0000
CMSSW           CMSSW_10_2_X_2018-04-15-1100 /cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc700/cms/cmssw-patch/CMSSW_10_2_X_2018-04-15-1100
...
```

**Pull request merged**